src/Tasks/Common/Resources/xlf/Strings.de.xlf

```diff
@@ -4,745 +4,805 @@
     <body>
       <trans-unit id="AddResourceWithNonIntegerResource">
         <source>NETSDK1076: AddResource can only be used with integer resource types.</source>
-        <target state="translated">NETSDK1076: AddResource kann nur mit ganzzahligen Ressourcentypen verwendet werden.</target>
+        <target state="new">NETSDK1076: AddResource can only be used with integer resource types.</target>
```

Comment:
Something went wrong in this file.

Reply:
I have no idea why it dropped all the German strings. I ended up reverting the file to its original content and just selectively injecting the new untranslated strings. I have no idea what happens the next time someone tries to add a resource string.

Reply:
OK, so after the manual restoration it managed to add new strings fine without dropping all the translation so it hopefully was just a one-off fluke local to my box.